Source/WebCore/rendering/svg/RenderSVGContainer.cpp

```diff
@@ -188,7 +188,14 @@ bool RenderSVGContainer::nodeAtPoint(const HitTestRequest& request, HitTestResul
             return true;
     }
 
-    // Spec: Only graphical elements can be targeted by the mouse, period.
+    // pointer-events=boundingBox makes it possible for containers to be direct targets
```

Comment:
Nit: https://webkit.org/code-style-guidelines/#comments-sentences
```suggestion
    // pointer-events=bounding-box makes it possible for containers to be direct targets.
```

Reply:
Done. Thanks @aprotyas - TIL.